overlay_top_strip.py

```diff
@@ -103,13 +93,13 @@ def draw_speed_max_speed(self):
         )
 
         # Actual speed (no colour change)
-        speed = self.data["gps_speed"].get()
         if self.data["gps_speed"].is_valid():
+            speed = self.data["gps_speed"].get()
             speed_text = "{0}".format(int(speed))
```

Comment:
Just need to add back in the `S: ` from before!
![image](https://user-images.githubusercontent.com/17876556/104711819-f2ba0700-5775-11eb-8463-619888f97ff6.png)